### PR TITLE
Update runtime

### PR DESCRIPTION
**Pull request by lionirdeadman**

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
bot, build

File: io.github.trytonvanmeer.DungeonJournal.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "io.github.trytonvanmeer.DungeonJournal",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.32",
+    "runtime-version": "40",
```
Comment (hfiguiere) [inline io.github.trytonvanmeer.DungeonJournal.json:4]: upgrading runtime requires the "dconf" migration.
See https://docs.flatpak.org/en/latest/sandbox-permissions.html#dconf-access

Comment (lionirdeadman) [reply io.github.trytonvanmeer.DungeonJournal.json:4]: Good catch, fixed

**Comment by lionirdeadman** [comment]
bot, build

**Pull request merged**